epic_resource_model

```diff
@@ -0,0 +1 @@
SXZxPOkqhwXn04FGHlsIxTkfpmGtGHVrKUYdHH04kZJrBXfutQq2Myt7DtW9JutLtgKP7+btwwFkDJzLuKUEUbInHKUFODctqSJHJRJIA21kC+rQRYYTDmR2JeNHu8zcVhY7oOFTdMYxe25e4ewwj05V2maMQVPt2ZI6gkN1hZA6d6tqrbx3DN9oPj8F+6JlTBaexPeEib20GRJsXB0iMOBLTzHDn6HIPyW1b84rjnrtoVA7kVargBNl0sa9xkS1JNULGvoLWFsRtdhymxFs+5kSb4crm2d9Hw9VQM2xyXCzcKdZmuLJ5lkm54aq5AwuOx+U5nclBwE1CtdbEBta68S4dEQvZAB0R+9kuHY2icvCFxDkeiSsrtRVXLusX/DbgeEDwab1WILk1beiYlmG19r4eoLKtr049NVQJZzB4qZEVK6QDCrvMsAveFI+u3aQcXcWjZBzDAl1bwUb8ZoCuPbAlZE/58bsWBTBOndb5atPpXsY171B2e0/ZcC/ajwYWeO60fcKa2cbKygLP9dsnrDF9DSXw3CnVK9u2AM3Si7VP06l/i+LrqOoO2KyJ9HqdbQAtNTamdRw8rcNtmdEti1O5qQymaKR1tIsGDMg3vPIACF79yHBpOWkg3A+CDENcv62jBHxRYk5GNmCl+B9q9eqkFkQsbrpD0mSnz8ujEvHKkLwcqGvIKGKgfo7dbJvBNVFk3UnnWWDb5GPpd4mRBbUnOsyJvLVutSHVcGvdYX05zASO5sTdZ3iiKM61mRMdkbul25xYKQrUlFhTHfiGwFqT6k95YyK4d2xMxPmOjRuPeVWIya4eCprxnLXeq4BjC+StAo22QHoIe20dXCZoSCjXHDyiCEtjFfzCmed4+8Z9zqFsYxLUyVe1vHY4JAMRGhFJcke/ZFnuQfwUvGS4nzXArwa+GXLvTZn0jh7pE9v2eo6hM7kxuZ2RLZNtHJ7gAHb484E+CNsacSDUYOAzx1gMCBz1v11OvW+0zIdljn09wNPnsF3byjgg17HOHFacjEjsCJCYpYTE+AUZ9LGPKIj93SAxedhDCv6zjVIgPz9cMXqEW4O8RRPlYCUAVKNS0JR9vcAPb3lpTMR3FFwrI/jZeYZ5ifjcKdYHfkQFSlTBT7FFsxiI7t0yKEQ7YK4o99jGR/v32AZN+/fYeGfXSDgqdIboRqnZe01xoBRXbGqEUqxPpzWro0Z7eI3W86YqGi3vvHPb0afRikqjvB+NFoXpQlzeefZOGnV+D+eDfP85egYhx/cwkJqLVuI6iL/f3XT2RemunrtVcEpHVqQC4eeozQ3jRtGUwr3Ua95a1+lbUCgzPqrXu88wz7fAm2Jt6vWE5zxSVK/XpbbQUK6ry12qda1x67NiXzWleu++aup0/75GeabM7RzFl8vt2vTeyZFm6y+DEN/Q4OmUvMTbVmygY/snO8cGNg8OKjJ5p+7JYVNvyxgWedyg8zrJt77TivWzJjvHDfqlJi/sS3f67eQ9pfxPjirlnwrCutSRShL8+kBYwtYVym3Ef//crja0baFoVV3cJFiJQlP2E+T5S/GikfeuJcu4hUG664IoCq5BxYIANO8dFey363l7iqkLw63IIz0Kvefu1IN/GnuQaTb3INstQ9YPtQdR753CHjTwGUWiFBa/vW/YnnnqtDpSz9fxDIXudPO9ywMhb3YnOqpT0l3WPY0jZ38/sWMwp1tNZUWtupRiVNympRO7E+fA8shSFkevdQjNxGvJD2PnQ/X8suiVsC/tgoiFh2841WXomkV7SiLWrfAHfDub0RN1DJ4ox26vTZM1XfIrwOorZO2Kslzk5WOUSrqsACqjPEYJyeRI7xQJLqPoK7jiKeiXoxdj5wlWLWmvzYnh7ZYggbcX6rogew3fYUEI8c9OauKSUKUmTgdVajjez0QcKZQ2DsKQEBc1iPm3bpKM8kyV3VUP/aBSRgPDW3KBdryIbUhA9Dowx5sy0MTfFeawqpSpeNpm/xIpwlvprRjgvafTPMlaPBYU5R1pvaT+Ds6mGdmf0ErDOxU/KQ03f5PebevQlmVX2btpvVJpYdNvC4t2Voy6jDPanlKmr8GqokxoiZ9KNFuvIePeU00n76gsRR7ruP6S5NEcvkCEkG5LbD9WMD58BWPXUTgBjRv+rjGn0+OV3FcyHLEVetsVP7YP32Q0mzQhRSulodIq9UNCMIsyyPcSshQspFWXa6vosuh1kIQ8tJjSBM1JFfHcaUYbq3bKxN/+SXka7W0Khf1He9vOM8wtcPyqSXmeyPDebXDj7NLUmMXbwPnGOF4nnGs9+iVmu+H7JVa2AfodZeTgK0gxBU9fJLKv4JP4DYB6A6QjnLAHSLe8MuaYHgBkHjoKWbl2qYjjhkbpVsukZ7e15B4g3ZN0pTvcwuf5bd4qhvYHLm67p/hhPQj5EiU13j8FiwUupRQmM7yJzaGmhsoTXSQrkjmOhoMrE8kKJInKFKGaR1HxG2Jm9KaDNfw5B/lsyelmidJscHklBLCLc6qRSpFgsGHkSzixC07LqgxTsA7hi8IVnmblc2yg3EjCU051pqoIlEFeC64FWVWJeorwO8aVp2gZqHYni6Q+tKok1Kf2AHgvjihMjdFGYYZLSfG1gd+nyJPn0DAruhs7nCpPFi8xiLEiDrggpuz2e/iQ/Gpz7FW9+wBh+b7iRmkJiiO8ktyjSTZHOs2NST6Vhb8fwN+qGoBqC7xirzXNbNjmEUzzPgkSR88a9FQrOWsINOXXYg0OzhpZQytrUKwrZQ1fEHnzWqzxJrKDjVPHdWp8hDIZ9oLlhzo4xFIG36LitYNnO7TDgJjgZvVsh/IkLWFfQ7nI/QmVeaU3vshrE4+OaWIVB913NKHpi4Immhzx0/eUxjw9qjTmKetGvUyDOeKKq9trBJYrDzxB4MbZ7TX8+TkJkUngkEi2Nln4ZCSlSZFWoGXDXk7awNYsifZFtvBjmiR5fT2lwWZJXq599V8=</diagram></mxfile>
```

Comment:
Is this the data model diagram?

Reply:
Yep. I moved that into the docs repo so it can be removed from this one